### PR TITLE
Create models and db

### PR DESCRIPTION
**Pull request by tomhill991**
Created db with postgresql and created all the models

### CODE REVIEW

File: app/models/message.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Message < ApplicationRecord
+  belongs_to :plant_chat
```
Comment (kaki-xue) [inline app/models/message.rb]: pls add has_many to Plant_chat model

File: app/models/plant.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Plant < ApplicationRecord
+  belongs_to :user
```
Comment (kaki-xue) [inline app/models/plant.rb]: user model - has_many :plants dependent: :destroy

File: app/models/message.rb (hunk 1 above)
Comment (kaki-xue) [reply app/models/message.rb]: PlantChat model - has_many :messages dependent: :destroy

File: app/models/plant_chat.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class PlantChat < ApplicationRecord
+  belongs_to :user
```
Comment (kaki-xue) [inline app/models/plant_chat.rb]: same has many dependent relationship

**Comment by tomhill991** [comment]
Made changes

**Pull request merged**